packages/protocol/scripts/truffle/make-release.ts

```diff
@@ -107,6 +112,77 @@ module.exports = async (callback: (error?: any) => number) => {
       throw new Error('All contracts linking libraries should be upgraded in release 1')
     }
 
+    const deployImplementation = async (contractName: string, Contract: any) => {
```

Comment:
If possible I would suggest for these helper methods to move outside of the exported function, otherwise it makes following this function quite challenging